scio-examples/src/main/scala/com/spotify/scio/examples/extra/AnnoyIndexSaveExample.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.ContextAndArgs
+import com.spotify.scio.extra.annoy._
+import com.spotify.scio.values.SCollection
+
+/**
+ * sbt "project scio-examples" "runMain com.spotify.scio.examples.extra.AnnoyIndexSaveExample \
```

Comment:
Afair other examples don't use `\`, as much as yours is more "correct", would be nice to stick with one convention?